astropy/cosmology/io/tests/test_mapping.py

```diff
@@ -14,6 +14,9 @@ class CosmologyWithKwargs(Cosmology):
     def __init__(self, name="cosmology with kwargs", meta=None, **kwargs):
         super().__init__(name=name, meta=meta, **kwargs)
 
+    def __equiv__(self, other):
```

Comment:
This makes me wonder whether it is really useful to make this an `abstractmethod`. Could the default just call `__eq__`? (Which is allowed to return `NotImplemented` as well)

Reply:
I actually think the default should be what I implement for FLRW. Cosmology has no parameters, but any subclass will, so that equivalency check will be for 1) the same class and 2) the same parameters.
I refactored ``__eq__`` to use ``__equiv__`` to minimize code duplication. It always uses ``Cosmology.__equiv__``,  because ``self.__equiv__`` is overwritten in subclasses, like ``FlatLambdaCDM``